medusa/post_processor.py

```diff
@@ -506,10 +506,20 @@ def symlink(cur_file_path, new_file_path):
                 self.log(u'Unable to link file {0} to {1}: {2!r}'.format
                          (cur_file_path, new_file_path, e), logger.ERROR)
                 raise EpisodePostProcessingFailedException('Unable to move and link the files to their new home')
+        
```

Comment:
@Kriskras99 The build is failing due to the white spaces here.

Reply:
@sharkykh thanks for the pointer